python/triton/experimental/gluon/language/amd/cdna3/__init__.py

```diff
@@ -0,0 +1,28 @@
+from ..._core import builtin
+from triton._C.libtriton import ir
+
+__all__ = ["buffer_load_to_shared"]
+
+
+@builtin
+def buffer_load_to_shared(dest, ptr, offsets, mask=None, other=None, stride=None, cache_modifier="", _semantic=None):
```

Comment:
Let's expose `stride` later--it's too detailed and we should think about how to properly expose it.